test/includes/check.sh

```diff
@@ -1,5 +1,23 @@
 # Miscellanous test checks.
 
+
+check_dependencies() {
+    # shellcheck disable=SC2039
+    local dep missing
+    missing=""
+
+    for dep in "$@"; do
+        if ! which "$dep" >/dev/null 2>&1; then
+            [ "$missing" ] && missing="$missing $dep" || missing="$dep"
+        fi
+    done
+
+    if [ "$missing" ]; then
+       echo "Missing dependencies: $missing" >&2
+       exit 1
+    fi
+}
```

Comment:
Can you move the check_dependencies part to the commit which introduces check.sh?